library/std/src/sys_common/fs.rs

```diff
@@ -11,7 +11,7 @@ pub fn copy(from: &Path, to: &Path) -> io::Result<u64> {
     if !metadata.is_file() {
         return Err(Error::new_const(
             ErrorKind::InvalidInput,
-            &"the source path is not an existing regular file",
+            &"the source path is neither a regular file or a symlink to a regular file",
```

Comment:
Not sure why it's duplicated like this, do we have a common place to store this sort of errors?

Reply:
The `sys` and `sys_common` structure is complicated and I'm not sure if we could achieve it straightforwardly.